framework/templates/ecl-filters/ecl-filters.twig

```diff
@@ -0,0 +1,149 @@
+{#
+  This is an example and should not be used as it is.
+#}
+
+<div class="ecl-row">
+  <aside class="ecl-col-lg-3" role="complementary">
+      <h3 class="ecl-heading ecl-heading--h3 ecl-u-mb-s">Filter by</h3>
+      <form action="#" method="get" id="{{ formID }}">
+        <div class="ecl-form-group">
+          {% include '@ec-europa/ecl-forms-labels' with { 'label': 'Keyword', 'for_attribute': 'filter-keywords' } %}
+          {% include '@ec-europa/ecl-forms-text-inputs' with {
+            'type': 'text',
+            'name': 'keywords',
+            'id': 'filter-keywords',
+            'extraAttributes': [
+              { 'value': '', 'size': '30', 'maxlength': '128' }
+            ]
+          } %}
+        </div>
+        <div class="ecl-form-group">
+          {% include '@ec-europa/ecl-forms-labels' with { 'label': 'Topic', 'for_attribute': 'filter-topic' } %}
+          {% embed '@ec-europa/ecl-forms-selects' with { 'name': 'topic', 'id': 'filter-topic' } %}
```

Comment:
An issue that's more related to the select, I'm ok to handle separately:

![filters europa component library](https://user-images.githubusercontent.com/1923476/28365929-cfd7684c-6c8a-11e7-9ed2-a4ba16f4ae1f.png)